src/Concerns/Api/ExtensionsTrait.php

```diff
@@ -131,4 +134,231 @@ public function updateUserExtensions(string $method = null, string $parameter =
+     *
+     * @return Result
+     */
+    public function createExtensionSecret(array $parameters = []): Result
+    {
+        $this->validateRequired($parameters, ['extension_id']);
+
+        return $this->post('extensions/jwt/secrets', $parameters);
+    }
+
+    /**
+     * Sends a specified chat message to a specified channel.
+     * The message will appear in the channel’s chat as a normal message.
+     * The “username” of the message is the Extension name.
+     *
+     * There is a limit of 12 messages per minute, per channel.
+     * Extension chat messages use the same rate-limiting functionality as the Twitch API (see Rate Limits).
+     *
+     * @see https://dev.twitch.tv/docs/api/reference#send-extension-chat-message
+     *
+     * @param array<string, mixed> $parameters
+     * @param array<string, mixed> $body
+     *
+     * @return Result
+     */
+    public function sendExtensionChatMessage(array $parameters = [], array $body = []): Result
+    {
+        $this->validateRequired($parameters, ['broadcaster_id']);
+        $this->validateRequired($body, ['text', 'extension_id', 'extension_version']);
+
+        return $this->post('extensions/chat', $parameters, null, $body);
+    }
+
+    /**
+     * Gets information about your Extensions; either the current version or a specified version.
+     *
+     * @see https://dev.twitch.tv/docs/api/reference#get-extensions
+     *
+     * @param array<string, mixed> $parameters
+     *
+     * @return Result
+     */
+    public function getExtensions(array $parameters = []): Result
+    {
+        $this->validateRequired($parameters, ['extension_id']);
+
+        return $this->get('extensions', $parameters);
+    }
+
+    /**
+     * Gets information about a released Extension; either the current version or a specified version.
+     *
+     * @see https://dev.twitch.tv/docs/api/reference#get-released-extensions
+     *
+     * @param array<string, mixed> $parameters
+     *
+     * @return Result
+     */
+    public function getReleasedExtensions(array $parameters = []): Result
+    {
+        $this->validateRequired($parameters, ['extension_id']);
+
+        return $this->get('extensions/released', $parameters);
+    }
+
+    /**
+     * Gets a list of Bits products that belongs to an Extension.
+     *
+     * @see https://dev.twitch.tv/docs/api/reference#get-extension-bits-products
+     *
+     * @param array<string, mixed> $parameters
+     *
+     * @return Result
+     */
+    public function getExtensionBitsProducts(array $parameters = []): Result
+    {
+        return $this->get('bits/extensions', $parameters);
+    }
+
+    /**
+     * Add or update a Bits products that belongs to an Extension.
+     *
+     * @see https://dev.twitch.tv/docs/api/reference#update-extension-bits-product
+     *
+     * @param array<string, mixed> $body
+     *
+     * @return Result
+     */
+    public function updateExtensionBitsProduct(array $body = []): Result
```

Comment:
I'd like to keep the query parameters argument present in case anything changes in the future.

```suggestion
    public function updateExtensionBitsProduct(array $parameters = [], array $body = []): Result
```